bootstrapvz/common/assets/init.d/expand-root

```diff
@@ -13,8 +13,19 @@
 prog=$(basename $0)
 logger="logger -t $prog"
 
-device_path="/dev/xvda"
+growpart=$(which growpart) || {
+	$logger "growpart was not found on PATH. Unable to expand size."
+	exit 0
+}
 
+root_index="0"
+
+$growpart /dev/xvda $root_index || {
+	$logger "growpart failed. Unable to expand size."
+	exit 0
+}
+
+device_path="/dev/xvda$root_index"
```

Comment:
`root_index` is set above here -- did you envision `device_path` as a whole being configurable outside of this init script? Or just moving `/dev/xvda` to another variable here?